### PR TITLE
[#176038005] method history

### PR DESCRIPTION
**Pull request by vito80ba**
Now there is a specific component to call the REST service to display the "history" of a payment method

### CODE REVIEW

File: .env.local [hunk 1]
```diff
@@ -1 +1,5 @@
-REACT_APP_API_URL=http://localhost
\ No newline at end of file
+REACT_APP_API_URL=http://localhost
```
Comment (gunzip) [inline .env.local:1]: of course this must change before merging :-)

File: azure-pipelines.yml [hunk 2]
```diff
@@ -30,6 +30,10 @@ variables:
   YARN_CACHE_FOLDER: $(Pipeline.Workspace)/.yarn
   BLOB_CONTAINER_NAME: '$web'
   REACT_APP_API_URL: 'https://api.io.italia.it'
+  ADB2C_TENANT_AUDIENCE: 'https://iobackoffice.b2clogin.com/iobackoffice.onmicrosoft.com/c2c9dbf8-9fc3-4f69-b8a6-c87d10d0ab06'
+  ADB2C_TENANT_AUTHORITY: 'https://iobackoffice.b2clogin.com/iobackoffice.onmicrosoft.com/B2C_1_backoffice'
+  ADB2C_TENANT_B2CSCOPES: 'https://iobackoffice.onmicrosoft.com/c2c9dbf8-9fc3-4f69-b8a6-c87d10d0ab06/ProfileRead'
+  ADB2C_TENANT_CLIENTID: 'c2c9dbf8-9fc3-4f69-b8a6-c87d10d0ab06'
```
Comment (gunzip) [inline azure-pipelines.yml:36]: can we keep these ones as pipelines variables ? in order to avoid to commit them

Comment (gunzip) [reply azure-pipelines.yml:36]: these changes are unrelated btw and must go in another pr

File: src/helpers/msal.ts [hunk 3]
```diff
@@ -19,15 +21,11 @@ interface IMsalConfig {
...
-  b2cScopes: [
-    "https://iobackoffice.onmicrosoft.com/c2c9dbf8-9fc3-4f69-b8a6-c87d10d0ab06/ProfileRead"
-  ],
+  audience: process.env.ADB2C_TENANT_AUDIENCE || "",
+  authority: process.env.ADB2C_TENANT_AUTHORITY || "",
+  b2cScopes: b2cscopes,
   changePasswordLink: "",
-  clientID: "c2c9dbf8-9fc3-4f69-b8a6-c87d10d0ab06",
+  clientID: process.env.ADB2C_TENANT_CLIENTID || "",
   redirectUri: window.location.origin,
```
Comment (gunzip) [inline src/helpers/msal.ts:29]: move this into another pr

**Review by gunzip — CHANGES_REQUESTED** [review]

File: .env.local (hunk 1 above)
Comment (vito80ba) [reply .env.local:1]: env.local file is for "local development"... where is the problem? :)

File: azure-pipelines.yml (hunk 2 above)
Comment (vito80ba) [reply azure-pipelines.yml:36]: These changes are yet in another PR (rebase when I made this one)
https://github.com/pagopa/backoffice.io.italia.it/pull/28 (waiting for approval)

**Review by pasqualedevita — COMMENTED** [review]
Can we add also `channel` and `channel_descr`?
https://github.com/pagopa/io-functions-backoffice/pull/33
`channel_descr` must to be visible on frontend
`channel` visible only in raw mode

**Pull request merged**